packages/@okta/vuepress-site/docs/guides/revoke-tokens/revokeatrt/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Revoke an access token or a refresh token
+---
+
+The token revocation endpoint can revoke either access or refresh tokens. Revoking an access token doesn't revoke the associated refresh token, though revoking a refresh token does revoke the associated access token.
+
+### Revoke only the access token
+
+Revoking only the access token effectively forces the use of the refresh token to retrieve a new access token. This could be useful if, for example, you have changed a user's data, and you want this information to be reflected in a new access token.
```

Comment:
I think this needs clarity on **what** is being forced:

> Revoking only the access token effectively forces [X] to use the refresh token to retrieve a new access token.

Also remove last comma:

>  This could be useful if, for example, you have changed a user's data and you want this information to be reflected in a new access token.

Reply:
This is a compound sentence and requires a comma before and